main/src/test/java/org/apache/james/jdkim/tagvalue/TagValueTest.java

```diff
@@ -73,7 +73,7 @@ public void testInvalidFWS() {
     @Test
     public void testInvalidFWSSyntax() {
         try {
-            new TagValue("p=test \r\n\r\n ");
+            new TagValue("p=test \r\n\r\n");
```

Comment:
I'm not familiar with FWS. But does the assertion still make sense once the whitespace after the CRLF is removed ?

Reply:
According to https://tools.ietf.org/html/rfc6376#section-2.8 , CRLF followed by at least one whitespace, so this new version without at least one whitespace after CRLF will be wrong FWS.

Reply:
And this is a invalid FWS test

Reply:
Thank you